.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,13 @@
-*Please check the [CONTRIBUTING.md](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) 
-file for relevant information on contributing to the documentation.*
 
-Thank you very much for your contribution!
+---
+
+> *Please check the [CONTRIBUTING.md](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) file for relevant information on contributing to the documentation.* Thank you for your contribution!
+
+Here's a list of commands supported by VersionBot
+
+- `@balena-ci I self-certify!` - use [the self-certification flow](https://docs.google.com/drawings/d/1aoXCdi8bzinxaLM0BoKMJaiZjIJj5Az02Npw_L2iQkE/edit?usp=sharing) for this PR
+- `@balena-ci test` - trigger the tests for this PR
+- `@balena-ci version` - generate the versioned branch for your PR
+- `@balena-ci rebase` - rebase the PR
+
+One can use these commands to manage, rebase or merge their own PR's as and when needed. We recommend using these commands to merge the PR if it's ready to be merge. 
```

Comment:
"to be merge." -> "to be merged."

Presumably these commands can only be used by Balena employees, rather than by 3rd-party users who might be contributing e.g. a "fix typo" PR?

Reply:
Yes, AFAIK these commands can only be used by balenistas. I was thinking to get more views on this or add a header that says only for balena Employees. I feel this is helpful for folks who don't know about versionbot. What do you think? As an external contributor does this create more confusion?

Reply:
Yeah, as an external contributor it _does_ feel like the PR template is the wrong place to be documenting @balena-ci commands :confused:

Reply:
Thanks for the feedback. I actually received some feedback from some folks internally as well. I think I will swap the location to a [balenista only] section at the bottom of the contributing guide to make it clear and helpful of the who, what, why of this feature.